src/pages/LandingPage.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { TrendingUp, Lightbulb } from "lucide-react";
+
+const Card: React.FC<{
```

Comment:
NB: Should this be pulled into its own component? Are we likely to re-use it elsewhere later (SBS)?

Reply:
See #482